### ISSUE TITLE
Can't set sdk path

### ISSUE THREAD

**Issue by Hercle999** [LibreOffice/loeclipse#99]
It couldn't set SDK path.

An loopexception (loopexception) occurred when set LibreOffice SDK path and push "OK" button on SDK Configuration for Libre office plugin. 
(ref. attached images)

I tried on some version's eclipse and on 2 PC, but same exception occurred.

![loopexception sdk](https://user-images.githubusercontent.com/56905781/226249354-2ba61d4a-584b-40b9-841c-1d74456acac2.png)
![stacktrace](https://user-images.githubusercontent.com/56905781/226249363-6989aba1-90d1-429b-89b8-da3fbd74d1fc.png)
![select sdk](https://user-images.githubusercontent.com/56905781/226249372-68858d68-31b4-4f35-a059-688efa2719ad.png)
![sdkconf](https://user-images.githubusercontent.com/56905781/226250088-dc8be2c1-e0cb-4f3e-b792-cf9363d8e3af.png)

**Comment by RayCulp** [comment]
@Hercle999 I found the problem. 

Starting with version 7.5.0.1, the SDK appears to be missing the entire IDL folder. 

Eclipse will not recognize the SDK if this folder is missing.

Workaround (which will most likely cause "unexpected behavior" downstream): 

1. In addition to the 7.5.X.X version of the SDK that is causing the problem, install version 7.4.6.2 of the SDK (the last version that included the IDL folder) for your operating system from here: https://downloadarchive.documentfoundation.org/libreoffice/old/7.4.6.2/
2. After installation completes successfully, copy the IDL folder and its contents from the 7.4.6.2 SDK installation folder over to the 7.5.X.X SDK folder.

![image](https://user-images.githubusercontent.com/7621330/226607251-6b1263fc-860d-453a-b938-9bd21dc61dcd.png)

Eclipse will now recognize the SDK.

Be aware that I have detected changes in files in the IDL folder from version to version, so this may cause errors downstream when you build your solution, until the LibreOffice devs fix the SDK and provide a new download.
